frontend/package.json

```diff
@@ -9,6 +9,8 @@
   },
   "dependencies": {
     "@mdi/font": "7.0.96",
+    "axios": "^1.6.5",
```

Comment:
Not an issue but a question: does frontend need proxying in the dev mode to avoid CORS issues?

Reply:
I am not sure - if I recall correctly we have some minimal CORS configuration in the customer project.
I can check tomorrow when setting up Oauth

Reply:
I am also not fully sure but I would say yes - it is more about browser security features, not related directly to the code and envs. However, I am guessing here